Grid grid
bool rightGripperOpen
bool leftGripperOpen
